webhooks/assign-class-label/webhook-config.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration 
+metadata:
+  name: assign-class-label-webhook
+webhooks:
+- name: assign-class-label-webhook.nerc.com
+  clientConfig: 
+    service:
+      namespace: rhods-notebooks
+      name: assign-class-label-webhook
+      path: /mutate
+    caBundle: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUVRRENDQXlpZ0F3SUJBZ0lVU3JnWEl4OUpON21wY3FnLzRsMWg5bFZNMFBNd0RRWUpLb1pJaHZjTkFRRUwKQlFBd2daQXhDekFKQmdOVkJBWVRBbFZUTVJZd0ZBWURWUVFJREExTllYTnpZV05vZFhOelpYUnpNUTh3RFFZRApWUVFIREFaQ2IzTjBiMjR4RURBT0JnTlZCQW9NQjFKbFpDQklZWFF4RVRBUEJnTlZCQXNNQ0ZKbGMyVmhjbU5vCk1ROHdEUVlEVlFRRERBWkpjMkZwWVdneElqQWdCZ2txaGtpRzl3MEJDUUVXRTJsemRHRndiR1YwUUhKbFpHaGgKZEM1amIyMHdIaGNOTWpRd09ERTVNVGN4TnpReFdoY05NamN3TmpBNU1UY3hOelF4V2pDQmtERUxNQWtHQTFVRQpCaE1DVlZNeEZqQVVCZ05WQkFnTURVMWhjM05oWTJoMWMzTmxkSE14RHpBTkJnTlZCQWNNQmtKdmMzUnZiakVRCk1BNEdBMVVFQ2d3SFVtVmtJRWhoZERFUk1BOEdBMVVFQ3d3SVVtVnpaV0Z5WTJneER6QU5CZ05WQkFNTUJrbHoKWVdsaGFERWlNQ0FHQ1NxR1NJYjNEUUVKQVJZVGFYTjBZWEJzWlhSQWNtVmthR0YwTG1OdmJUQ0NBU0l3RFFZSgpLb1pJaHZjTkFRRUJCUUFEZ2dFUEFEQ0NBUW9DZ2dFQkFJbEJnaXZMVVZWWENmOTIyV2JHN3Fwcm9MZUgvODlQCkNVbHJLSTFISUNYRW1tZTRKRHJvQzRIV1c0QW4vS1VoVW5pNkpWbHhjaE02dWJMeU5sNjBmRVRERWJBeHgzWVUKa1RHa01JNGxWS0VlVGlpcFJVQUdwQjg0VXBEbi82cnpwQUJqT2xoakFmMUNZcnhzSVJvc1o2L1ZVcG9veTdOZApiYllIdlFTMmd1N2IwODAyS2YrWko2bUpPbGlIcUdtVjRqVlRJK0RPem15RXlhMmhwSXpHMnZOQ0JIcDJVZXN2CkdaMXJ0dE8xOWs3K1AyQXdoZFVUMk14Z2NEeUdaSjF5N2ZuR25JbXczRGtEL2FKWUp3bklHaWhaMnNSZTlsbjgKbGVsRVN3YWx3S2NPTlZRcnB3QlJzdC8ySXdhdlpyd1dNQVg0cUpEeks3dWJyc3FJdkZZcUh6MENBd0VBQWFPQgpqekNCakRBZEJnTlZIUTRFRmdRVXpDWUswU2tkbm02S0lNbnFiSGF1QnlSOWxrb3dId1lEVlIwakJCZ3dGb0FVCnpDWUswU2tkbm02S0lNbnFiSGF1QnlSOWxrb3dEd1lEVlIwVEFRSC9CQVV3QXdFQi96QTVCZ05WSFJFRU1qQXcKZ2k1aGMzTnBaMjR0WTJ4aGMzTXRiR0ZpWld3dGQyVmlhRzl2YXk1eWFHOWtjeTF1YjNSbFltOXZhM011YzNaagpNQTBHQ1NxR1NJYjNEUUVCQ3dVQUE0SUJBUUI3eVpGM1k5ZWFkTDR5Y1R3cHJsem5JT2IreUJncGxkOFdlNndqCisvWE9XVnlXb09rM1JYT0VadWoxUE8xcWpHY0lJOFpQc05xamJmNVhOd1hkaUlsSm1KdDJaQVFWWkpqY0dkMmQKS2RRNXF1Wk44Z2ZFS0pwWHJUVWlIZ3VQQjdCNmd5dGVBYWlyaXl4RDllZmgzcG1HTGJaU0xscXlRcVBrUFZ6UwpUeXBIbDNPVHllaE8wKy82MWh0MnE1TVpYamd4Rk1CQTlqNXZKd2JUMlVxWFllSGFOOTlqcU0rTW10Zks2NUUxCjkxQUY2bndmQys0Z1FNS0tUQ0hpVkFWb09KakJBVjZyOXkydSs5a0xyNE9LUkFaWEF4alI3dFl1YitsQmJYZUsKSHpOdnBYajBLZmpBaG9BNkdPUW1MVG01UGRNZjN2dWhlS1FFdmp1L3lldzZTMlk5Ci0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0K
```

Comment:
You should not need to provide this here; from the docs:

```
$ kubectl explain MutatingWebhookConfiguration.webhooks.clientConfig.caBundle
GROUP:      admissionregistration.k8s.io
KIND:       MutatingWebhookConfiguration
VERSION:    v1

FIELD: caBundle <string>

DESCRIPTION:
    `caBundle` is a PEM encoded CA bundle which will be used to validate the
    webhook's server certificate. If unspecified, system trust roots on the
    apiserver are used.
```

The "system trust roots on the apiserver" should be sufficient.

I bring this up because we would need a different `caBundle` on every server on which this ever gets deployed, whereas if we can remove it from the manifest, we can use the same manifest everywhere.